### DISCUSSION TITLE
Tiny plugin view

### DISCUSSION THREAD

**Discussion by lostinthegrove** [apohl79/audiogridder#488] [Help] [answered]
This might not be the right place for this. Audio Gridder works perfectly but when I unplug my display on my slave machine(mac mini late 2014) the plugin view becomes tiny and I cant see the full plugin. Anyone have any advice?

**Comment by AtmanActive** [comment, marked as answer]
I think you need this: 

https://www.google.com/search?q=dummy+hdmi+dongle&oq=dummy+hdmi&aqs=chrome.2.69i57j0l5j0i395l4.4727j1j7&sourceid=chrome&ie=UTF-8

**Reply by lostinthegrove** [reply]
Thanks will check it out
